diff --git a/node_modules/rtf-parser/rtf-interpreter.js b/node_modules/rtf-parser/rtf-interpreter.js
index 09a89dd..bf4b5b0 100644
--- a/node_modules/rtf-parser/rtf-interpreter.js
+++ b/node_modules/rtf-parser/rtf-interpreter.js
@@ -185,11 +185,7 @@ class RTFInterpreter extends Writable {
     this.group.style.italic = set !== 0
   }
   ctrl$u (num) {
-    var charBuf = Buffer.alloc ? Buffer.alloc(2) : new Buffer(2)
-    // RTF, for reasons, represents unicode characters as signed integers
-    // thus managing to match literally no one.
-    charBuf.writeInt16LE(num, 0)
-    this.group.addContent(new RTFSpan({value: iconv.decode(charBuf, 'ucs2')}))
+    this.group.addContent(new RTFSpan({value: String.fromCodePoint([num])}))
   }
   ctrl$super () {
     this.group.style.valign = 'super'
